elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -152,7 +153,7 @@ class PfeTabs extends PFElement {
     super.connectedCallback();
 
     this.addEventListener("keydown", this._onKeyDown);
-    this.addEventListener("click", this._onClick);
+    // this.addEventListener("click", this._onClick);
```

Comment:
Does this need to be uncommented?  Do we not want to use this click event?

Reply:
@castastrophe @kylebuch8 I have uncommented that line.